docs/index.md

```diff
@@ -0,0 +1,320 @@
+
+**lolex** can be used to simulate passing time in automated tests and other
+situations where you want the scheduling semantics, but don't want to actually
+wait (however, from version 2.0 **lolex** supports those of you who would like to wait too).
+
+**lolex** is extracted from [Sinon.JS](https://github.com/sinonjs/sinon.js).
+
+## Installation
+
+```shell
+npm install lolex --save-dev
+```
+
+You are always free to [build it yourself](https://github.com/sinonjs/lolex/blob/53ea4d9b9e5bcff53cc7c9755dc9aa340368cf1c/package.json#L22), of course.
+
+## Usage
+
+**lolex** can be used in both Node and browser environments!
+
+In addition in browser environment **lolex** provides a [`performance`](#performancenow) implementation that gets its time from the clock.
+In Node environments **lolex** provides a [`nextTick`](#nexttick) implementation that is synchronized with the clock - and a [`process.hrtime`](#hrtimeprevtime) shim that works with the clock.
+
+If you want to use **lolex** in a browser you can use
+[the pre-built version](https://github.com/sinonjs/lolex/blob/master/lolex.js)
+available in the repo and the npm package.
+
+Using npm you only need to reference **lolex** in your `<script>` tags.
+
+```html
+<script src="./node_modules/lolex/lolex.js">
+...
+</script>
+```
+
+To use **lolex**, create a new clock, schedule events on it using the timer
+functions and pass time using the [`tick`](#tick) method.
+
+```js
+// In the browser distribution, a global lolex is already available
+var lolex = require("lolex");
+
+var clock = lolex.createClock();
+
+clock.setTimeout(function () {
+    console.log("The poblano is a mild chili pepper originating in the state of Puebla, Mexico.");
+}, 15);
+
+// ...
+
+clock.tick(15);
+```
+
+Upon executing the last line, an interesting fact about the
+[Poblano](http://en.wikipedia.org/wiki/Poblano) will be printed synchronously to
+the screen. If you want to simulate asynchronous behavior, you have to use your
+imagination when calling the various functions.
+
+The [`next`](#next), [`runAll`](#runall), [`runToFrame`](#runtoframe), and [`runToLast`](#runtolast) methods are available to advance the clock.
+See the [clock API Reference](#clock-api) for more details.
+
+### Faking the native timers
+
+When using **lolex** to test timers, you will most likely want to replace the native
+timers such that calling `setTimeout` actually schedules a callback with your
+clock instance, not the browser's internals.
+
+Calling [`install`](#installconfig) with no arguments achieves this. You can call `uninstall`
+later to restore things as they were again.
+
+```js
+// In the browser distribution, a global lolex is already available
+var lolex = require("lolex");
+
+var clock = lolex.install();
+// Equivalent to
+// var clock = lolex.install(typeof global !== "undefined" ? global : window);
+
+setTimeout(fn, 15); // Schedules with clock.setTimeout
```

Comment:
Add `clock.tick(15); // fn is called` maybe?

Reply:
Ping @mgred ?

Reply:
I add the line: https://github.com/sinonjs/lolex/pull/221/commits/67723ca7366b2b14d9a75ddcf4b9ce5f6d87d582#diff-1a523bd9fa0dbf998008b37579210e12R101
Sorry, what is wrong here?
Thanks

Reply:
Ah sorry, you didn't resolve it (in the GitHub UI) so I assumed there was still more work :) I'll resolve it.